rio-config/src/defaults.rs

```diff
@@ -95,6 +95,12 @@ cursor = '▇'
 #
 blinking_cursor = true
 
+# Ignore theme selection foreground color
+#
+# Default is false
+#
+ignore_theme_selection_fg_color = false
```

Comment:
I think the default is true no? https://github.com/raphamorim/rio/pull/227/files#diff-b64bebdc21e5973e8c9ff4e193b6533793ef05ec19c6c812fbcf02be5367dafaR360

Reply:
Yes, that is a mistake from my side. 

The default should of course be false, so it doesn't break existing behavior. 

I have just pushed a fix.

Reply:
perfect! 🚀